cmd/bnsd/app/app.go

```diff
@@ -118,19 +122,43 @@ func Stack(issuer weave.Address, minFee coin.Coin) weave.Handler {
 	return Chain(authFn, minFee).WithHandler(Router(authFn, issuer))
 }
 
+// CronStack wires up a standard router with a cron specific decorator chain.
+// This can be passed into BaseApp.
+// Cron stack configuration is a subset of the main stack. It is using the same
+// components but not all functionalities are needed or expected (ie no message
+// fee).
+func CronStack() weave.Handler {
+	authFn := cron.Authenticator{}
+	decorators := app.ChainDecorators(
+		utils.NewLogging(),
+		utils.NewRecovery(),
```

Comment:
Ticker implementation is [managing the transaction manually](https://github.com/iov-one/weave/blob/e7395104c19aacc362d179dd6431fb30fabaf593/x/cron/cron.go#L184) to ensure consistency and atomic changes.
Should I add this anyway to ensure one more save point on top?

Reply:
You don't have to. Either is fine. The use of cache after Discard in the `tick` implementation is not good (we cannot rely on any behavior after Discard) and I thought this would make it simpler. You can do the same manually as well.. whichever is clearer for you